tests/unit/backend/modules/bounce/bounce.controller.spec.ts

```diff
@@ -0,0 +1,59 @@
+import HttpStatus from 'http-status-codes'
+import { mocked } from 'ts-jest/utils'
+
+import handleSns from 'src/app/modules/bounce/bounce.controller'
+import * as bounceService from 'src/app/modules/bounce/bounce.service'
+
+jest.mock('src/app/modules/bounce/bounce.service')
+const mockBounceService = mocked(bounceService, true)
+
+describe('handleSns', () => {
+  let req, res
+  beforeEach(() => {
+    req = { body: 'somebody' }
+    res = { sendStatus: jest.fn() }
+  })
+  afterEach(() => {
+    mockBounceService.updateBounces.mockReset()
+    mockBounceService.isValidSnsRequest.mockReset()
+  })
+  it('does not call updateBounces for invalid requests', async () => {
```

Comment:
Should we follow the test naming conventions too?

```
it('should <x> when <y>
```

I know, "should" makes it sound like its optional, but imagine it as "we don't know if the test will pass, but it _should_ pass, and if it does not, it means the something is broken". (Also in contractual contexts like in tests/assertions _should_ means _must_ so there's that angle)

```suggestion
  it('should not call updateBounces when sns request is invalid', async () => {
```

Reply:
For all the tests in this file